gettsim/interface.py

```diff
@@ -117,6 +122,43 @@ def compute_taxes_and_transfers(
     return results
 
 
+def _fail_if_datatype_is_false(data, columns_overriding_functions, functions):
+    """Check if the provided data has the right types.
+
+    Parameters
+    ----------
+    data : pandas.Series or pandas.DataFrame or dict of pandas.Series
+        Data provided by the user.
+    columns_overriding_functions : str list of str
+        Names of columns in the data which are preferred over function defined in the
+        tax and transfer system.
+    functions : dict of callable
+        A dictionary of functions.
+
+    Returns
+    -------
+    ValueError
+        Fail if the data types are not matching the required in gettsim.
+
+    """
+    check_data = True
```

Comment:
That needs to go in the for-loop if I am not mistaken

Reply:
I think it doesn't matter. If once check_data is set to False it stops and raises the error....Oh well maybe for the debug mode. Changed it!